native/cocos/platform/mac/modules/Screen.mm

```diff
@@ -44,7 +44,15 @@ of this software and associated engine source code (the "Software"), a limited,
 }
 
 float Screen::getDevicePixelRatio() const {
+#if CC_EDITOR
+    se::AutoHandleScope hs;
+    se::Value ret;
+    char commandBuf[100] = "window.devicePixelRatio";
+    se::ScriptEngine::getInstance()->evalString(commandBuf, 100, &ret);
```

Comment:
Don't use `evalString` since it's expensive than `getProperty`.
```suggestion
auto* global = se::ScriptEngine::getInstance()->getGlobalObject();
se::Value devicePixelRatioVal;
global->getProperty("devicePixelRatio", &devicePixelRatioVal);
return devicePixelRatioVal.isNumber() ? devicePixelRatioVal.toFloat() : 1.F;
```